src/main/java/ch/njol/skript/variables/MySQLStorage.java

```diff
@@ -0,0 +1,55 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.variables;
+
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.log.SkriptLogger;
+import lib.PatPeter.SQLibrary.MySQL;
```

Comment:
Is this library still functional on modern (My)SQL versions? Last commit was in 2014 https://github.com/PatPeter/SQLibrary

It might be better to switch to a different library, one that is actually updated and more popular

Reply:
Not targeting this pull request. This pull request opens the ability for addon to do exactly that.